WordPress/src/main/res/values/strings.xml

```diff
@@ -935,6 +935,7 @@
     <string name="error_refresh_unauthorized_comments">You don\'t have permission to view or edit comments</string>
     <string name="error_refresh_unauthorized_pages">You don\'t have permission to view or edit pages</string>
     <string name="error_refresh_unauthorized_posts">You don\'t have permission to view or edit posts</string>
+    <string name="error_refresh_profile_not_accessible">There was an error trying to access your profile information</string>
```

Comment:
Looks like it got collapsed due to an "outdated diff" - https://github.com/wordpress-mobile/WordPress-Android/pull/3698#discussion_r51639762